src/NuGet.Core/NuGet.Packaging/Rules/AnalysisResources.resx

```diff
@@ -213,6 +213,9 @@
   <data name="MisplacedTransformFileWarning" xml:space="preserve">
     <value>The transform file '{0}' is outside the 'content' folder and hence will not be transformed during installation of this package. Move it into the 'content' folder.</value>
   </data>
+  <data name="MissingReadmeInformation" xml:space="preserve">
+    <value>The created package is missing a readme. Go to https://learn.microsoft.com/en-us/nuget/create-packages/package-authoring-best-practices#readme to learn why package readmes are important.</value>
```

Comment:
Thanks for reviewing!
I'm not sure what would be the best url here, when people misses adding a readme file into the package.
Hi @JonDouglas @albarry4 , what do you think?

Reply:
I agree with aka.ms idea. 

> aka.ms/nugetauthoringbestpractices

Reply:
Thanks! Changed the URL into aka.ms/nuget/authoring-best-practices#readme

Reply:
Out of curiosity, did you try putting the `#readme` in the aka.ms url's redirection, rather than in the URL we output to customers?

I'm thinking just in case the markdown document for the docs changes `## README` to something else, the `#readme` is the URL will no longer be valid. If we can make it part of the redirect URL (I have no idea if that works), then we'll be able to maintain it by changing the redirection URL.

Reply:
I created a Aka link [nuget/authoring-best-practices/readme](https://aka.ms/nuget/authoring-best-practices/readme) for https://learn.microsoft.com/en-us/nuget/create-packages/package-authoring-best-practices#readme. Changed `#` into `/`, as `#` is not allowed in the Aka link.
Looks like there is a benefit to track the [traffic data](https://microsoft.sharepoint.com/teams/RedirectionServiceInfo/SitePages/Aka-Link-Manager-Tool-FAQ.aspx#traffic-data) for this link. Thanks for the suggestion!